packages/core/admin/admin/src/pages/Admin/Onboarding/index.js

```diff
@@ -62,6 +63,8 @@ const StyledLink = styled.a`
   }
 `;
 
+library.add(faFile, faBook);
```

Comment:
Why do you need to add these two and not `faQuestion` or `faTimes`? 🤔

Reply:
You can do it in your test file directly at runtime it's not an issue because FA is loaded.

FA is so huge in the bundle we need to be careful